src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,32 @@
+package duke;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    private ArrayList<Task> taskList;
+
+    public TaskList() {
+        this.taskList = new ArrayList<Task>();
```

Comment:
I like how you state the type (`Task`) explicitly, instead of shorten it to `new ArrayList<>()`.